apps/browser/qml/pages/components/TabView.qml

```diff
@@ -203,10 +200,95 @@ Item {
 
         ]
     }
+
     TabsToolBar {
         id: tabsToolBar
         anchors.bottom: parent.bottom
         onBack: pageStack.pop()
-        onEnterNewTabUrl: root.enterNewTabUrl()
+        onEnterNewTabUrl: tabView.enterNewTabUrl()
+        onOpenMenu: popupMenu.active = true
+    }
+
+    PopUpMenu {
+        id: popupMenu
+
+        signal closeAllTabs
+
+        anchors.fill: parent
+        active: false
+
+        menuItem: Component {
+            Item {
+                id: menuItem_
+                readonly property int iconWidth: Theme.iconSizeMedium + Theme.paddingLarge
```

Comment:
minor
```suggestion
                readonly property real iconWidth: Theme.iconSizeMedium + Theme.paddingLarge
```

Reply:
I think a uniform spelling is better 
https://github.com/sailfishos/sailfish-browser/blob/next/apps/browser/qml/pages/components/PopUpMenuItem.qml#L16